app/src/main/res/values/strings.xml

```diff
@@ -1,22 +1,22 @@
 <resources>
     <string name="app_name" translatable="false">Vigilante</string>
     <string name="vigilante_description">Vigilante service</string>
-    <string name="vigilante_summary">Vigilante service listener for events that displays dot when camera/mic usage happens.</string>
+    <string name="vigilante_summary">Listens for events and displays a dot when camera/mic usage occurs.</string>
```

Comment:
Vigilante Listens

Reply:
Thought it went without saying with the title, but forgot where it is found.